### DISCUSSION TITLE
Why do all the downstream turbines have the same power output, except for the first turbine?

### DISCUSSION THREAD

**Discussion by binson910718** [OpenFAST/openfast#2588] [General]
I am currently running a case study with four wind turbines arranged in a row, using the IEA Wind 15-MW Wind Turbine. The spacing between each turbine is 7 rotor diameters (diameter = 240 meters). The results are shown in the figure below. Normally, the power output of the turbines should decrease further downstream. Why are the power outputs of the third and fourth turbines almost the same as the second turbine? What might be causing this issue?
![Same power](https://github.com/user-attachments/assets/79607864-c853-431d-9b5d-4140c58c1a10)

**Comment by jjonkman** [comment]
Dear @binson910718,

I would say your expectation is not a general rule, and the power down the row will depend on the specific conditions considered.  You've given little information on your model setup, but I'm not surprised by the results you are seeing.

Best regards,

**Reply by binson910718** [reply]
![image](https://github.com/user-attachments/assets/dd5d0612-840b-48a7-a9fe-741d5306d2f8)
![image](https://github.com/user-attachments/assets/f06ab758-1303-4c13-a765-13629937074a)
![image](https://github.com/user-attachments/assets/1cb21cef-6f3d-4076-b111-b1794e847724)
Here are all the settings in my FAST.Farm.fstf file. Are there any errors in the configuration?

**Reply by jjonkman** [reply]
Dear @binson910718,

Your settings look OK, except that:

- When using TurbSim for ambient wind, we generally recommend using `Mod_AmbWind` = 3 rather than 2.
- Your low-resolution domain width and height do not seem large enough to cover wake meandering (e.g., typically we recommend having a 3 times diameter buffer on the sides and top of the domain).

I presume you are using TurbSim, although you haven't said so explicitly.  What wind conditions have you set in your _InflowWind.dat_ file?

To ensure you are following all FAST.Farm modeling guidance (https://openfast.readthedocs.io/en/main/source/user/fast.farm/ModelGuidance.html), we generally recommend using the python scripts available in the OpenFAST Toolbox when defining the inflow and setting up the spatial-temporal resolutions used by FAST.Farm: https://github.com/OpenFAST/openfast_toolbox/tree/main/openfast_toolbox/fastfarm.

Best regards,

**Reply by binson910718** [reply]
![image](https://github.com/user-attachments/assets/d856f7c1-2fff-49c3-8f6c-1b59181d3d9e)
![image](https://github.com/user-attachments/assets/a4d20e86-4444-49b4-8771-618a66732dd4)
Here are my execution results. I used WindType is steady, and windspeed is 10.59m/s.

**Reply by jjonkman** [reply]
Dear @binson910718,

Thanks for clarifying.  While my previous suggestions don't apply when using steady wind, please note that the wake parameters of FAST.Farm are not well calibrated for steady wind.  The wake parameters likely need to be recalibrated to get accurate results in steady wind.

Best regards,

**Reply by binson910718** [reply]
Dear @jjonkman 

What if my windtype is set to uniform?

Best regards,

**Reply by jjonkman** [reply]
Dear @binson910718,

The original calibration of FAST.Farm wake parameters from which the DEFAULT parameters were set was based on turbulent wind inflow: https://www.nrel.gov/docs/fy18osti/70533.pdf.  The wake parameters likely need to be recalibrated to get accurate results in steady or uniform wind, which does not have full-field turbulence.

Best regards,

**Reply by binson910718** [reply]
Dear @jjonkman 

I found that besides FLORIS, there are also WindSE and SOWFA. For the case I want to calculate, which one would you recommend using?

Best regards,

**Reply by jjonkman** [reply]
Dear @binson910718,

If your goal is to calculate the steady-state power of a wind farm in uniform wind inflow conditions, FLORIS is NREL's popular tool for that.  WindSE could also be used.  I would say FAST.Farm and SOWFA would be overkill for that specific application.

Best regards,

**Reply by binson910718** [reply]
Dear @jjonkman

Thank you for your suggestion; I will give FLORIS a try.

Best regards,
